porthole/queries.py

```diff
@@ -34,6 +34,13 @@ def json_converter(obj):
     def as_dict(self):
         raise DeprecationWarning("QueryResult.as_dict method is no longer available and will be removed.")
 
+    def as_list_of_tuples(self):
```

Comment:
A list of tuples (or a list of lists) is the most sensible way to add data to a worksheet in openpyxl: https://openpyxl.readthedocs.io/en/stable/api/openpyxl.worksheet.worksheet.html#openpyxl.worksheet.worksheet.Worksheet.append.

So this operation will need to be performed at some point before the use of the openpyxl API.  If not in the queries API we could handle in `ReportReadWriter` in `components.py`.  Or we could go down one level and handle in `WorkbookEditor` in `xlsx.py`.  I just figured if I was going to make the method, we might as well put it somewhere where it could be reused.  But again, I don't feel strongly about it.  Where do you thing is a more appropriate location for this code?

Reply:
I would suggest removing the method and just handling the data as a list of lists, since that structure is already being used effectively. Based on your comment and the linked docs, it sounds like there's no specific benefit to using tuples.